src/components/core/radio/radio-group.tsx

```diff
@@ -0,0 +1,99 @@
+import * as Stitches from '@stitches/react';
+import { styled } from 'Styles/stitches.config';
+import { modifyVariantsForStory } from 'Styles/type-utils';
+import * as RadioGroupPrimitive from '@radix-ui/react-radio-group';
+
+type RadioGroupProps = {
+    dark?: boolean;
```

Comment:
I think as this is a boolean prop, it can be `is_dark`.

Reply:
@hamid-deriv kept it dark to be consistent with other components in deriv-ui where is dark is used for dark mode